sources/packages/backend/apps/api/src/services/disbursement-schedule/_tests_/unit/disbursement-schedule.getMaxTuitionRemittance.spec.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  NotificationActionsService,
+  SequenceControlService,
+} from "@sims/services";
+import { DisbursementScheduleService } from "../../disbursement-schedule-service";
+import { DataSource } from "typeorm";
+import { addDays } from "@sims/utilities";
+import { COE_WINDOW } from "../../../../utilities";
+import { COEApprovalPeriodStatus } from "../../disbursement-schedule.models";
+
+describe("DisbursementScheduleService-getCOEApprovalPeriodStatus", () => {
+  let service: DisbursementScheduleService;
+
+  beforeAll(async () => {
+    const dataSource = {} as DataSource;
+    dataSource.getRepository = jest.fn();
+    const sequenceService: SequenceControlService =
```

Comment:
No need for explicit type declarations.
It can be simplified to:
```ts
const sequenceService = {} as SequenceControlService;
const notificationActionsService = {} as NotificationActionsService;
```